packages/fxa-content-server/server/bin/fxa-content-server.js

```diff
@@ -120,6 +120,11 @@ function makeApp() {
     })
   );
 
+  app.use(
+    helmet.referrerPolicy({
+      policy: 'no-referrer',
```

Comment:
Couple of comments:
* It looks like this is [the default](https://helmetjs.github.io/#referrer-policy) setting. Is this overwritten somewhere, wherein setting it here properly sets it?
* I assume elements that have the `rel` attribute explicitly set won't be impacted

This seems fine to me to test out though!

One other comment or rather my $0.02:

> When navigating from getpocket.com, the window.location.search params are empty after the request https://accounts.firefox.com/signup?hello=123 is loaded. Content-server at this point is not aware of the params and fails to load the correct OAuth clients

This makes me wonder if it's possible that there's some delay in setting and validating values on content-server reliers and being "ready" when we need to use them, but take that with a grain of salt since I haven't looked at this issue.

Reply:
> This makes me wonder if it's possible that there's some delay in setting and validating values on content-server reliers

In the Safari debugger, the Backbone app hasn't been loaded yet but `windows.location.search` is still empty string. This is a bit of a shot in the dark but I think low risk to try.

> It looks like this is [the default](https://helmetjs.github.io/#referrer-policy) setting

Ah, yea I think it is the default if you use `app.use(helmet());`, but if you use the `referrerPolicy` directly you have to specify it.